charts/anyscale-operator/values.yaml

```diff
@@ -45,9 +45,9 @@ operatorReplicas: 1
 # Node selectors that apply specifically to these instance types can be
 # placed directly under each instance type under the `nodeSelector` key.
 defaultInstanceTypes:
-  2CPU-8GB:
+  4CPU-8GB:
     resources:
-      CPU: 2
+      CPU: 4
```

Comment:
we should be careful about breaking change as mentioned in https://anyscaleteam.slack.com/archives/C068KQ0GAFJ/p1738089559382939

Reply:
what lead to this change?